### PR TITLE
Bump nodejs to latest 18.x

### PR DESCRIPTION
**Pull request by kachick**
Node18 looks enough production ready atleast for mobu.

Note
---

Initally I tried to replace jest with node18 builtin test runner, however it does not care typescript yet. So I have omitted from this PR.

https://github.com/nodejs/node/pull/42325

### CODE REVIEW

**Comment by kachick** [comment]
Netlify build is falling with https://lavaux.lv/2022/04/26/fix-netlify-build-errors-with-node-18.html 👀 

```
2:42:01 PM: node: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by node)
node: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.25' not found (required by node)
node: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by node)
```

**Comment by kachick** [comment]
@pankona Could you check and update the Netly image?

>Site settings > Build & deploy > Continuous Deployment > Build image selection.

**Comment by pankona** [comment]
@kachick 

I've changed the Build image selection from Ubuntu 16.04 to Ubuntu 20.04.
Could you try it again?

**Comment by kachick** [comment]
@pankona Thank you! Looks working ✨

**Comment by kachick** [comment]
I have updated actions and branch protection rule, it is a blocker 😤

File: package.json [hunk 1]
```diff
@@ -12,9 +12,9 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^14.2.1",
-    "@tsconfig/node16-strictest-esm": "^1.0.3",
+    "@tsconfig/node18-strictest-esm": "^1.0.0",
```
Comment (kachick) [inline package.json:15]: @pankona 

FYI: This npm was born for mobu 👶

**Comment by kachick** [comment]
>Initally I tried to replace jest with node18 builtin test runner, however it does not care typescript yet. So I have omitted from this PR.

https://github.com/meyfa/ts-node-test make it can.

**Pull request merged**